MediaGallerySynchronization/Model/AssetImagesSynchronization.php

```diff
@@ -75,10 +75,7 @@ public function __construct(
     }
 
     /**
-     * Create MediaGallery asset and save it to database based on file information
-     *
-     * @param \SplFileInfo $item
-     * @throws \Exception
+     * @inheritDoc
      */
     public function execute(\SplFileInfo $item): void
```

Comment:
Please change the method to work with multiple `$items` (\SplFileInfo[])
```suggestion
    public function execute(array $items): void
```